zebedee-cms/src/main/java/com/github/onsdigital/zebedee/util/SlackNotification.java

```diff
@@ -27,88 +43,163 @@
+            attch.fields.add(new PostMessageField("Approval status", collection.getDescription().approvalStatus.toString(), true));
+        }
 
-            msg.append("\nTransaction IDs:");
-            c.getDescription().publishTransactionIds.entrySet()
-                    .stream().forEach(e -> msg.append("\n - " + e.getValue()));
+        if (collection.getDescription().publishTransactionIds != null
+                && !collection.getDescription().publishTransactionIds.isEmpty()) {
+            attch.fields.add(new PostMessageField("Transaction IDs", StringUtils.join(collection.getDescription().publishTransactionIds.values(), "\n"), true));
         }
-        alarm(msg.toString());
     }
 
-    public static void alarm(String message) {
-        String slackUsername = "Alarm";
-        String slackEmoji = ":heavy_exclamation_mark:";
-        send(message, slackToken, slackAlarmChannel, slackUsername, slackEmoji);
-    }
+    private static void addCollectionAttachment(PostMessageAttachment attch, CollectionBase collection) {
+        attch.fields.add(new PostMessageField("Collection", "<" + Configuration.getFlorenceUrl() + "/florence/collections/" + collection.getId() + "|" + collection.getName() + ">", true));
 
-    private static void sendPublishNotification(String message) {
-        String slackUsername = "Bot";
-        String slackEmoji = ":chart_with_upwards_trend:";
-        send(message, slackToken, slackPublishChannel, slackUsername, slackEmoji);
-    }
+        if(collection.getPublishDate() == null) {
+            attch.fields.add(new PostMessageField("Publish date", "Manual", true));
+        } else {
+            attch.fields.add(new PostMessageField("Publish date", slackFieldFormatVague.format(collection.getPublishDate()), true));
+        }
 
-    public static void send(String message) {
-        String slackUsername = "Bot";
-        String slackEmoji = ":chart_with_upwards_trend:";
-        send(message, slackToken, slackDefaultChannel, slackUsername, slackEmoji);
-    }
+        if(collection.getClass() == PublishedCollection.class) {
+            PublishedCollection publishedCollection = (PublishedCollection)collection;
 
-    public static void send(String message, String slackToken, String slackChannel, String slackUsername, String slackEmoji) {
+            if (publishedCollection.publishStartDate != null && publishedCollection.publishEndDate != null) {
+                String timeTaken = String.format("%.2f", (publishedCollection.publishEndDate.getTime()
+                        - publishedCollection.publishStartDate.getTime()) / 1000.0);
 
-        if (slackToken == null || slackChannel == null) {
-            return;
-        }
+                attch.fields.add(new PostMessageField("Duration", timeTaken + " seconds", true));
+                attch.fields.add(new PostMessageField("Publish start date", slackFieldFormatAccurate.format(publishedCollection.publishStartDate), true));
```

Comment:
I would consider making "Publish start data" and the other Strings you are using a couple of times constants.